### PR TITLE
Fix several minor fine overview issues

### LINKED ISSUE #118: Fine issues
- [x] While applying the dates on fine overview, some users' names don't appear. I suspect this is because of GewisUserResponse and UserResponse conflicts.
- [x] Selection should disappear after notify or hand out.
- [x] There is no feedback on notify.
- [x] When opening the datepicker for the reference dates, the time defaults to the current minute, but resets to 12 AM

### PR DESCRIPTION
**Pull request by CodeNamedRobin**
Fixes #118.

### CODE REVIEW

File: src/components/Mutations/MutationsTableComponent.vue [hunk 1]
```diff
@@ -100,7 +100,6 @@ import {
   parseTransaction,
   parseTransfer
 } from "@/utils/mutationUtils";
-import { formatPrice } from "@/utils/formatterUtils";
```
Comment (SuperVK) [inline src/components/Mutations/MutationsTableComponent.vue:103]: Even though the IDE says this is unused, it is in fact in the <template> and removing this break everything:
image

File: src/views/FineView.vue [hunk 2]
```diff
@@ -268,14 +290,38 @@ const handoutFines = async () => {
...
+        detail: t('successMessages.finesHandedOut'),
+        life: 3000,
+        severity: 'success',
+      });
+    })
+    .catch((err: AxiosError) => handleError(err, toast))
+    .finally(() => selection.value = []);
+};
+
+// NOTE: This is a temporary fix, until PrimeVue 3.52, where this issue should be fixed in the library
```
Comment (SuperVK) [inline src/views/FineView.vue]: I propose we do not do this fix, just upgrade when 3.52 releases. It is not that much of an issue and I expect 3.52 to be released before 22 may.

File: src/views/FineView.vue [hunk 3]
```diff
@@ -254,10 +264,22 @@ const openHandoutEvent = async (eventId: number) => {
 };
 
 const notifyUsers = async () => {
+  isNotifying.value = true;
   await apiService.debtor.notifyAboutFutureFines({
     userIds: selection.value.map((item: any) => item.id),
     referenceDate: secondDate.value?.toISOString() || new Date().toISOString()
-  });
+  })
```
Comment (SuperVK) [inline src/views/FineView.vue:269]: Why are you both awaiting, and using .then()? https://dev.to/masteringjs/using-then-vs-async-await-in-javascript-2pma

File: src/views/FineView.vue [hunk 4]
```diff
@@ -191,20 +196,25 @@ const handlePickedDates = handleSubmit(async (values) => {
...
       ...item,
+      // @ts-ignore
+      gewisId: (userStore.users.find((u: UserResponse) => u.id === item.id) as UserResponse).gewisId || undefined,
       fullName,
       // Assign first and last balance based on the first and second items in the balances array
       firstBalance,
       lastBalance: secondBalance,
     };
-  });
+  }).filter((u: any) => !deletedUsers.includes(u.id) && activeUsers.includes(u.id));
```
Comment (SuperVK) [inline src/views/FineView.vue:215]: Why is this not fixed in the backend but in the frontend?

**Review by SuperVK — CHANGES_REQUESTED** [review]

File: src/views/FineView.vue (hunk 4 above)
Comment (CodeNamedRobin) [reply src/views/FineView.vue:215]: I want to do both, and for now the bug is not fixed in the backend yet.

**Review by SuperVK — APPROVED** [review]
LGTM!

**Pull request merged**